packages/ckeditor5-coremedia-content/src/commands/OpenInTabCommand.ts

```diff
@@ -62,27 +62,25 @@ export class OpenInTabCommand extends Command {
     // might not update correctly once displayed. You will have to trigger #refresh
     // manually in order to display the correct content state
     // (e.g. of a suddenly unreadable content).
-    serviceAgent
-      .fetchService<WorkAreaService>(new WorkAreaServiceDescriptor())
-      .then((workAreaService: WorkAreaService): void => {
-        workAreaService
-          .canBeOpenedInTab([uriPath])
-          .then((canBeOpened: unknown) => {
-            logger.debug("May be opened in tab: ", canBeOpened);
-            this.isEnabled = canBeOpened as boolean;
-          })
-          .catch((error): void => {
-            logger.warn(error);
-            this.isEnabled = false;
-          });
-      });
+    serviceAgent.fetchService(createWorkAreaServiceDescriptor()).then((workAreaService: WorkAreaService): void => {
```

Comment:
```suggestion
    serviceAgent.fetchService(createWorkAreaServiceDescriptor()).then((workAreaService): void => {
```

Explicit typing should be obsolete, now.